docs/maintainers/generic-how-to-implement-a-concept-exercise.md

```diff
@@ -27,7 +27,9 @@ languages
 
 ## Step 1: add .docs/introduction.md
 
-This file contains an introduction to the concept. It should be explicit about what the student should learn from the exercise, and provide a short, concise introduction to the concept(s). The aim is to give the student just enough context to figure things out themselves and solve the exercise, as research has shown that self-discovery is the most effective learning experience. Mentioning technical terms that the student can Google if they so want, is preferable over including any code samples or an extensive description. For example we might describe a string as a "Sequence of Unicode characters" or a "series of bytes" or "an object". Unless the student needs to understand the details of what those mean to be able to solve the exercise we should not give more info in this introduction - instead allowing the student to Google, ignore, or map their existing knowledge.
+This file contains an introduction to the concept. It should be explicit about what the student should learn from the exercise, and provide a short, concise introduction to the concept(s). The aim is to give the student just enough context to figure things out themselves and solve the exercise, as research has shown that self-discovery is the most effective learning experience. Mentioning technical terms, that the student can Google if they so want, is preferable over an extensive description. The exception to this is syntax, which we don't want the student to need Google, so code samples should be included whenever syntax is introduced.
```

Comment:
Why is this text repeated?  Wouldn't it be better for maintenance to refer to the previous explanation?

Reply:
The reason for this is simple: we want people to be able to copy the contents from this file into their tracks, so it must be in this file. That said, the information also must be in the other file, as that described the documents.

The above information is then copy-pasted into the track-specific version of the above file. Yes, this does mean duplication, but the benefit is that it becomes incredibly easy to just copy/paste content and get working. The track-specific version of the above content will be used in GitHub issues and we don't want to refer to an external document, which makes it less friendly to contribute.